contracts/instruments/RibbonThetaVault.sol

```diff
@@ -334,7 +381,7 @@ contract RibbonThetaVault is DSMath, OptionsVaultStorage {
         scheduledWithdrawals[msg.sender] = 0;
         queuedWithdrawShares = queuedWithdrawShares.sub(withdrawShares);
 
-        uint256 amountAfterFee = _withdraw(withdrawShares, true);
+        uint256 amountAfterFee = _withdraw(withdrawShares, true, false);
```

Comment:
Wasn't the scope of this PR, but let's remove the `withdrawLater` and `completeScheduledWithdrawal` functions from the vault because it's a v2 feature